components/raftstore/src/store/fsm/apply.rs

```diff
@@ -845,7 +844,7 @@ impl ApplyDelegate {
         if !data.is_empty() {
             let cmd = util::parse_data_at(data, index, &self.tag);
 
-            if should_write_to_engine(&cmd, apply_ctx.kv_wb().count()) {
+            if should_write_to_engine(&cmd) || apply_ctx.kv_wb().should_write_to_engine() {
```

Comment:
why not pass `apply_ctx.kv_wb()` into `should_write_to_engine`?

Reply:
Because I think it is two method...

Reply:
One is checked for cmd and another is checked for WriteBatch.